src/components/OnBoarding/VisitTracker.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from "@emotion/react";
+
+import Header from "@/components/common/Header/Header";
+
+import { theme } from "@/styles/theme";
+
+function VisitTracker() {
+	const discoveryRouteText = ["앱 스토어", "지인 추천", "SNS 광고", "검색", "기타"];
+
+	return (
+		<>
+			<Header>
+				<Header.PrevButton />
+			</Header>
+			<div css={visitTrackerStyle}>
+				<h1>어떤 경로를 통해 오셨나요?</h1>
+				<ul>
+					{discoveryRouteText.map((item) => (
+						<li>
```

Comment:
key prop이 없어서 콘솔에 경고창이 뜨는데 key 추가하는거 어떠세요? 
값 자체가 고정되어 있는거라 없어도 될거같긴 하지만 콘솔에 경고 떠있으면 나중에 에러 메시지 보기가 힘들더라구요
```suggestion
						<li key={item}>
```